editor/src/components/common/notices.tsx

```diff
@@ -26,6 +26,8 @@ export const getStylesForLevel = (level: NoticeLevel): React.CSSProperties => {
   return resultingStyle
 }
 
+const ToastTimeout = 5500
```

Comment:
I thought we already had a timeout for these?

Reply:
I removed the setTimeout in the ADD_TOAST update function, because this PR is making toasts inside the transient canvas state, and pushing the toasts to editorState.toasts only if the transient state is applied at the end of the interaction (we are not showing the toasts during the drag or if the user abandons). 

I actually like the timeout better in the Toast component as well, because it starts counting when the toast visually shows up, not when the ADD_TOAST is called for it.

Reply:
does this still preserve the distinction between toasts you need to close manually, and those that close themselves automatically?

Reply:
yep, the `persistent` property controls that

Reply:
https://github.com/concrete-utopia/utopia/blob/e998d9d9dab8ba6e865343b29862152d18de1d2a/editor/src/components/common/notices.tsx#L45